pkg/plugins/optional/helm/v1alpha/edit.go

```diff
@@ -18,12 +18,15 @@ package v1alpha
 
 import (
```

Comment:
Any change in the scaffolds requires we run `make install` and `make generate` could you please do it?
Also, please ensure that we have the commits squashed , we should have 1 commit, 1 fix/rfe per PR so that we can ensure a good git history and release notes.